templates/views/views-view--txex-events--block-1.html.twig

```diff
@@ -62,6 +62,8 @@
     {% set tempArray1 = tempArray1|merge({(key+1) : (child),}) %}
   {% endfor %}
 
+  {# Moving no results text inside the block instead of outside#}
+  {% set tempArray1 = {0 : empty}|merge(tempArray1) %}
   {% set tempArray1 = {0 : exposed}|merge(tempArray1) %}
```

Comment:
I'm not sure that this will work with the array key of '0' being used for both items. Maybe it does?

Reply:
@mikeethedude Can you please chime in here?

Reply:
It _works_. Perhaps we can approach this in a way that reads more clearly though.

Reply:
It's already an ugly way to do it (the existing pattern that you're innocently following). If it works, then no worries. Leave as is. Some day... all will be moved to preprocess? Today is definitely not that day. :)

Reply:
Sounds like a refactor for another day then. Thanks for talking through it with us.